pkg/operation/seed/seed.go

```diff
@@ -298,9 +325,155 @@ func BootstrapCluster(seed *Seed, k8sGardenClient kubernetes.Client, secrets map
+	certManagementConfig, err := certmanagement.RetrieveCertificateManagementConfig(certificateManagement)
+	if err != nil {
+		return nil, err
+	}
+
+	var (
+		clusterIssuerName = certManagementConfig.ClusterIssuerName
+		acmeConfig        = certManagementConfig.ACME
+		route53Config     = certManagementConfig.Providers.Route53
+		clouddnsConfig    = certManagementConfig.Providers.CloudDNS
+	)
+
+	var dnsProviders []certmanagement.DNSProviderConfig
+	for _, route53provider := range route53Config {
+		it := route53provider
+		dnsProviders = append(dnsProviders, &it)
+	}
+	for _, cloudDNSProvider := range clouddnsConfig {
+		it := cloudDNSProvider
+		dnsProviders = append(dnsProviders, &it)
+	}
+
+	var (
+		letsEncryptSecretName = "letsencrypt-secret"
+		secretAccesKey        = "accessKey"
+		secrets               = createDNSProviderSecrets(dnsProviders, secretAccesKey)
+		providers             = createDNSProviderValues(dnsProviders, secretAccesKey)
+		acmePrivateKey        = acmeConfig.ACMEPrivateKey()
+	)
+
+	if len(acmePrivateKey) > 0 {
+		letsEncryptKeySecret := &corev1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      letsEncryptSecretName,
+				Namespace: common.GardenNamespace,
+			},
+			Data: map[string][]byte{
+				"tls.key": []byte(acmePrivateKey),
+			},
+		}
+		secrets = append(secrets, letsEncryptKeySecret)
+	}
+
+	if err := createCertificateIssuerSecrets(k8sSeedclient, secrets); err != nil {
+		return nil, err
+	}
+
+	return map[string]interface{}{
+		"name": string(clusterIssuerName),
+		"acme": map[string]interface{}{
+			"email":               acmeConfig.Email,
+			"server":              acmeConfig.Server,
+			"privateKeySecretRef": letsEncryptSecretName,
+			"dns01": map[string]interface{}{
+				"providers": providers,
+			},
+		},
+	}, nil
+}
+
+func createCertificateIssuerSecrets(k8sSeedClient kubernetes.Client, clusterIssuerSecrets []*corev1.Secret) error {
+	for _, clusterIssuerSecret := range clusterIssuerSecrets {
+		secret, err := k8sSeedClient.CreateSecretObject(clusterIssuerSecret, true)
+		if err != nil {
+			return err
+		}
+		logger.Logger.Debugf("%s secret was created successfully", secret.Name)
+	}
+	return nil
+}
+
+func createDNSProviderValues(configs []certmanagement.DNSProviderConfig, secretAccessKey string) []interface{} {
+	var providers []interface{}
+	for _, config := range configs {
+		name := config.ProviderName()
+		switch config.DNSProvider() {
+		case certmanagement.Route53:
+			route53config, ok := config.(*certmanagement.Route53Config)
+			if !ok {
+				logger.Logger.Errorf("Failed to cast to Route53Config object for DNSProviderConfig  %+v", config)
+				return nil
+			}
+
+			providers = append(providers, map[string]interface{}{
+				"name":        name,
+				"type":        certmanagement.Route53,
+				"region":      route53config.Region,
+				"accessKeyID": route53config.AccessKeyID,
+				"secretName":  name,
```

Comment:
As said above: When you create the secrets via the Helm chart this name is not needed any more (it can be computed within the Helm chart).